base/src/commonMain/kotlin/utils/UnitSystem.kt

```diff
@@ -22,12 +22,14 @@ enum class UnitSystem {
     METRIC, MIXED, IMPERIAL;
 
     companion object {
+        // FIXME IMPERIAL and US customary seem reversed
         fun withRawValue(value: String): UnitSystem = when (value) {
             "U.S." -> IMPERIAL
             "U.K." -> MIXED
             else -> METRIC
         }
 
+        // FIXME IMPERIAL and US customary seem reversed
         fun withCountryCode(code: String): UnitSystem = when {
             listOf("GB", "MM", "LR").contains(code) -> MIXED
             "US" == code -> IMPERIAL
```

Comment:
Noticed this in existing code, it is reversed isn't it?

Reply:
No, the UK uses a mixed system (parts are UK imperial, others are metric)

Reply:
Actually, both UK and US use a mixed version of metric and imperial. So I understand @ChristoferAlexander 's confusion.

I suggest having 4 types: SI, METRIC (note: SI != metric system), UK_CUSTOMARY, US_CUSTOMARY.

Reply:
See the first paragraph of https://en.wikipedia.org/wiki/System_of_measurement#Imperial_and_US_customary_units (also referred to by Java Measurement API). They mention 4 types/systems but do not mention the UK Customary one, see also https://en.wikipedia.org/wiki/Metrication_in_the_United_Kingdom